server/app.js

```diff
@@ -1,25 +1,39 @@
-const express = require('express');
+const express = require('express')
+const mongo = require('mongodb');
 const path = require('path');
 
-const app = express();
 
+// path to database
+const db = require('../db/db.js')
 
-// configuration
-const port = 3000;
+// create server
+const app = express()
 
+// configuration
+const PORT = 3000
 
-//middleware
+// middleware
 app.use(express.json());
-app.use(express.static(path.join(__dirname, 'public'))
+app.use(express.static(path.join(__dirname, 'public')))
 
 
 // routes
-app.get('/', (req, res) => {
-  res.send('Hello Test!');
-});
-
-app.listen(port, () => {
-  console.log(`app listening on port${port}`)
-});
-
-// app basic functionality is up and running
\ No newline at end of file
+app.get('/people', (req, res) => {
+  db.models.people.find()
+  .exec()
+  .then((data) => {
+    res.send(data)
+  })
+  .catch((err) => {
+    console.log('you have an err', err)
+    res.end()
```

Comment:
I think that
```js
console.log('you have an err', err):
res.end();
```
should send the error to the front-end instead of creating a console log in the server's console. Something more along the lines of replacing it with the following:
```js
  .catch((err) => {
    res.status(400).send(err.stack);
```